src/core/meshcop/meshcop_tlvs.hpp

```diff
@@ -1495,15 +1495,17 @@ class ChannelMaskTlv : public Tlv
      * @returns A pointer to first Channel Mask Entry or NULL if not found.
      *
      */
-    const ChannelMaskEntry *GetFirstEntry(void) const;
+    const ChannelMaskBaseEntry *GetFirstEntry(void) const;
 
     /**
      * This method gets the Page 0 Channel Mask Entry in the Channel Mask TLV.
      *
+     * @param[in]  aChannelPage  The ChannelPage value.
+     *
      * @returns A pointer to Page 0 Channel Mask Entry or NULL if not found.
      *
      */
-    const ChannelMask0Entry *GetMask0Entry(void) const;
+    const ChannelMaskEntry *GetMaskEntry(uint8_t aChannelPage) const;
```

Comment:
As by default Thread only has channel page 0, how about letting `aChannelPage` here has default parameter 0 and also setting channel page as 0 in` Init()`?
Besides, for `ChannelTlv` in this header file, how about also set mChannelPage as 0 in 'Init()'?
Thus no need to set channel page explicitly for default thread behavior.
Thoughts?